src/renderer/components/chat/ChatListItem.tsx

```diff
@@ -116,7 +116,7 @@ function Message({
             }}
           />
         )}
-        <div>{message2React(summaryText2 || '', true)}</div>
+        {message2React(summaryText2 || '', true)}
```

Comment:
Was it necessary to make things work to remove the div?

Reply:
I essentially moved that inside message2React as `<div className='truncated'>`. I thought it's a part of preview logic.